scripts/qa_crowbarsetup.sh

```diff
@@ -4184,7 +4184,8 @@ function oncontroller_testsetup
 
     wait_for 40 5 "timeout -k 20 10 ssh -o UserKnownHostsFile=/dev/null $ssh_target true" "SSH key to be copied to VM"
 
-    if ! ssh $ssh_target curl $clouddatadns/test ; then
+    [[ $http_proxy ]] && http_proxy_env="http_proxy=$http_proxy" || http_proxy_env=""
+    if ! ssh $ssh_target $http_proxy_env curl -s --fail http://suse.com ; then
```

Comment:
In the old days, this was http://www3.zq1.de/test.txt and while that was sufficiently stable in practice, it was not so nice to rely on my private webserver being up ;-)